rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -51,7 +52,7 @@ object RholangCLI {
 
     val conf = new Conf(args)
 
-    val runtime = Runtime.create(conf.data_dir(), conf.map_size())
+    val runtime = Runtime.create(conf.data_dir(), conf.map_size())(Capture.taskCapture)
```

Comment:
`captureTask` argument to the `Runtime.create` is supposed to be an implicit but in all the places, except tests, we pass it explicitly. Let's stick to one approach and either delete the `implicit` part from the signature or pass it implicitly. What do you think? I would prefer to keep as much consistency as we can. This way the cognitive load is minimized.

Reply:
Yep, keeping them implicit is fine with me.